living/irish.md

```diff
@@ -5,41 +5,53 @@ Additional names:
 - 
 
 News:
-- 
-- 
+- https://www.irishtimes.com/gaeilge/tuarascail/
+- https://www.meoneile.ie/
+- https://www.rte.ie/gaeilge/ (Ireland's National Public Service Broadcaster)
 
 Culture / History:
-- 
-- 
+- https://www.duchas.ie/ga/ (The National Folklore Collection)
+- https://www.museum.ie/ga-ie/home (The National Museum of Ireland)
+- https://www.ireland.ie/ga/cuairt/ (Irish Tourism Site)
+- https://www.gov.ie/ga/an-roinn-cult%C3%BAir-cumars%C3%A1ide-agus-sp%C3%B3irt/ (Department of Culture, Communications and Sport)
```

Comment:
I think we could just add `https://www.gov.ie/ga/` here and the "An Roinn Cultúir, Cumarsáide agus Spóirt" page will be included